mlir/lib/Dialect/Rock/Transforms/GemmToGridwise.cpp

```diff
@@ -235,6 +266,127 @@ GemmRewritePattern::matchAndRewrite(GemmOp op, GemmOpAdaptor adaptor,
   return success();
 }
 
+std::tuple<Value, Value, Value>
+GemmRewritePattern::arrangeSplitKTransform(OpBuilder &builder, GemmOp op,
```

Comment:
Can we add a single test to demo this behaviour (mostly as documentation) in https://github.com/ROCm/rocMLIR/blob/develop/mlir/test/Dialect/Rock/gemm_to_gridwise.mlir ?

Reply:
Hi @manupak, here is the test:

https://github.com/ROCm/rocMLIR/blob/09b8348ed385c1c0f527a9738a0537c725cdbad6/mlir/test/Dialect/Rock/gemm_to_gridwise.mlir#L83-L99

`kpackPerBlock * kpack = 8`, `split-k = 3` and the original `K` length is equal to `238`. In this case, the correct padding should be `240`. The test confirms it